policy-controller/k8s/index/src/index.rs

```diff
@@ -1172,11 +1176,71 @@ impl PolicyIndex {
         authzs
     }
 
-    fn http_routes(&self, server_name: &str) -> HashMap<String, InboundHttpRoute> {
+    fn route_client_authzs(
+        &self,
+        route_name: &str,
+        authentications: &AuthenticationNsIndex,
+    ) -> HashMap<AuthorizationRef, ClientAuthorization> {
+        let mut authzs = HashMap::default();
+
+        for (name, spec) in self.authorization_policies.iter() {
```

Comment:
style nit, not important: `&HashMap` implements `IntoIterator`, so i believe this could just be 
```suggestion
        for (name, spec) in self.authorization_policies {
```